src/utils/spotify.js

```diff
@@ -39,9 +41,26 @@ const fetchSpotifyApiData = async (url, token) => {
   return data;
 };
 
+export const createPlaylist = async (token, userId, playlistName) => {
```

Comment:
Also make sure you create Private playlist by default not Public. (Public playlist visible to anyone, private - visible only for the owner)